z.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"go/types"
+
+	"golang.org/x/tools/go/packages"
+)
+
+type (
+	ZObject struct {
```

Comment:
Let's name it `ParsedType` or something similar.

Reply:
As discussed, use `ZType`